tools/update_vendor.sh

```diff
@@ -40,4 +40,4 @@ rm -fr "${KUBE_ROOT}/vendor/k8s.io/cloud-provider-gcp/providers"
 ln -s "../../../providers" "${KUBE_ROOT}/vendor/k8s.io/cloud-provider-gcp/providers"
 
 # restore BUILD files in vendor/
-bazel run //:gazelle
+bazelisk run //:gazelle
```

Comment:
why we do need to change this?

Reply:
oops, need to revert this one. In my desktop, I was not able to install Bazel @ 5.4.0. I finally was able to do that by downloading the installer artifact from the 5.4.0 release https://github.com/bazelbuild/bazel/releases/tag/5.4.0